src/Engine/EngineResult.php

```diff
@@ -35,4 +35,19 @@ public function getText(): string {
 	public function getWarnings(): array {
 		return $this->warnings;
 	}
+
+	/**
+	 * Normalize result by replacing some historic characters
+	 */
+	public function normalize() {
+		$this->text = strtr( $this->text, [
```

Comment:
Some (and more) of these translations could be done with [Normalizer::normalize( $this->text, Normalizer::FORM_KC)](https://www.php.net/manual/en/normalizer.normalize.php), but that causes a runtime conflict with the Symfony class which is also called Normalizer.

Reply:
> but that causes a runtime conflict with the Symfony class which is also called Normalizer.

It should work fine as long as you use `\Normalizer` here or `use Normalizer;` at the top, to use the intl extension's one. The Symfony class is a polyfill for that for when the intl extension isn't installed. If you use the former, then don't forget to add that extension to the requirements in `composer.json`.